regression/ansi-c/main1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE test-c++-front-end
+main.c
+
+'main' with signature 'signed int \(char \*argv, signed int arc\)' found
+^EXIT=(64|1)$
+^SIGNAL=0$
+--
+Invariant check failed
```

Comment:
Can we add a description of what the test is trying to achieve?  Maybe something like:
> In this test, we detect a non-standard C entry point and gracefully exit with a message to the user.

Reply:
Thank you, done!